### PR TITLE
130 points in just 30 seconds

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: 2021-22/2021-12-09_130p.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from math import sqrt
+
+def main():
+    drivetrain.set_drive_velocity(100,PERCENT)
+    fork_motor_group.spin_to_position(1800, DEGREES, wait=False)
+    goto( -920,  920, 0)
+    goto( -920,-1450, 0)
+    goto(  750,-1400, 0) # blue in right zone
+    goto(  500,-1150, 1)
+#   goto(    0, -950, 0) # m = - 2/5
```
Comment (kreier) [inline 2021-22/2021-12-09_130p.py:12]: Why is this commented line still in the code?

Comment (ssis-robotics) [reply 2021-22/2021-12-09_130p.py:12]: It's the exact location to pick up the yellow goal. But with the adjusted corresponding coordinates the robot drives in a straight line, picking up the goal on the way.

Comment (kreier) [reply 2021-22/2021-12-09_130p.py:12]: So m = - 2/5 is the slope or gradient of the linear motion? @ssis2021

Comment (ssis-robotics) [reply 2021-22/2021-12-09_130p.py:12]: correct.

**Pull request merged**